precompiles/pallet-democracy/src/lib.rs

```diff
@@ -284,8 +284,8 @@ where
 		})?;
 
 		log::trace!(target: "democracy-precompile",
-			"Delegating vote to {:?} with balance {:?} and {:?}",
-			representative, conviction, amount
+			"Delegating vote to {:?} with balance {:?} and conviction {:?}",
+			representative, amount, conviction,
```

Comment:
Note that you can format variables by writing them directly in the template :p

```suggestion
			"Delegating vote to {representative:?} with balance {amount:?} and conviction {conviction:?}",
```